prow/scripts/cluster-integration/reconciler-e2e-gardener.sh

```diff
@@ -82,6 +82,8 @@ readonly COMMON_NAME_PREFIX="grd"
 utils::generate_commonName -n "${COMMON_NAME_PREFIX}"
 
 export INPUT_CLUSTER_NAME="${utils_generate_commonName_return_commonName:?}"
+# This is needed for the gardener::cleanup function
+export CLUSTER_NAME="${INPUT_CLUSTER_NAME}"
```

Comment:
I suggest add this `CLUSTER_NAME` to `gardener::init()` as one of `requiredVars`